presto-docs/src/main/sphinx/release/release-0.192.rst

```diff
@@ -0,0 +1,81 @@
+=============
+Release 0.192
+=============
+
+General Changes
+---------------
+
+* Fix correctness issue in the :func:`geometry_to_bing_tiles` function that causes
+  it to return irrelevant tiles when bottom or right side of the bounding box of the
+  geometry is aligned with the tile border.
+* Fix handling of invalid WKT (well-known text) input in geospatial functions.
+* Fix an issue that can cause long-running queries to hang when writer scaling is enabled.
+* Fix cast from ``REAL`` or ``DOUBLE`` to ``DECIMAL`` to conform to the SQL standard.
+  For example, previously ``cast (double '100000000000000000000000000000000' as decimal(38))``
+  would return ``100000000000000005366162204393472``. Now it returns ``100000000000000000000000000000000``.
+* Fix incorrect reporting of input size and positions in live plan view.
+* Fix bug in validation of resource groups that prevented use of the ``WEIGHTED_FAIR`` policy.
+* Fix HTTP 500 errors from the coordinator due to upstream errors when fetching data from workers.
```

Comment:
Fail queries properly when the coordinator fails to fetch data from workers. Previously, it would return an HTTP 500 error to the client.